lib/trader-sdk/lib/indicators/bb-percent-b-mfi.ts

```diff
@@ -0,0 +1,71 @@
+import _ from 'lodash';
+import { BandWidth } from '../bollinger-bands';
+import { MFI } from '../money-flow-index';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/dills122/trader-tools/pullRequest?prid=8102010)
```suggestion
import { MFI } from "../money-flow-index";
```